Gruntfile.js

```diff
@@ -69,42 +28,13 @@ module.exports = function( grunt ) {
 				cache: "build/.sizecache.json"
 			}
 		},
-		tests: {
```

Comment:
Can we migrate other tasks as well? It can happen in a separate PR. Migrate should be on a similar maintenance level as Core so it'd be good to get rid of Grunt here as well.

Reply:
Yes, but that might come a good bit later. My first priority is getting jenkins decommissioned.